bcipy/helpers/convert.py

```diff
@@ -8,16 +8,23 @@
 import numpy as np
 from pyedflib import FILETYPE_EDFPLUS, EdfWriter
 
-from bcipy.helpers.load import load_json_parameters, read_data_csv
-from bcipy.helpers.triggers import read_triggers, trigger_durations
+from bcipy.helpers.load import load_json_parameters, read_data_csv, extract_mode
+from bcipy.helpers.triggers import trigger_decoder, apply_trigger_offset, trigger_durations
 
 
 def convert_to_edf(data_dir: str,
                    edf_path: str = None,
                    overwrite=False,
-                   use_event_durations=False) -> Path:
+                   write_targetness=True,
```

Comment:
Do you think it's a more common scenario that we would want to see the targetness rather than the stimuli? Should the default be False?